config/phobos.yml.example

```diff
@@ -1,6 +1,9 @@
 logger:
   # Optional log file, set to false or remove to disable it
   file: log/phobos.log
+  # Optional output format for stdout, default is false (human readable).
```

Comment:
I think this could be rephrased to make it easier to read. I think right now it's quite hard to understand. Maybe something like this:
```Optional set to true to enable json output to stdout, default is false (human readable).```
WDYT?